docs/usage/examples/self-hosting.md

```diff
@@ -1,4 +1,4 @@
-# Self-Hosting Renovate
+# Self-Hosting Examples
```

Comment:
@viceice suggested (https://github.com/renovatebot/renovate/discussions/8488#discussioncomment-1158676):

> @rarkins @HonkingGoose i think we should add our helm chart as preferred version for Kubernetes and remove the other Kubernetes samples.

@viceice I think this file would need to be changed then?

Reply:
Yes.